plugin-test/build.gradle

```diff
@@ -0,0 +1,53 @@
+/*
+    The MIT License (MIT)
+    
+    ${PROJECT_NAME} Copyright (c) ${YEAR} ${USER}
```

Comment:
Something strange happened with this header.

@06needhamt, I generally prefer to not put the headers like that in my source files. They need separate maintainership, so I personally prefer to just leave the `License.md` file at the repository root and not mention the license in the sources at all.

Could we do the same in this project? Or do you have any arguments for such headers (I tend to agree with you then)?